src/laplace.jl

```diff
@@ -31,7 +31,7 @@ using Parameters
     backend::Symbol = :EmpiricalFisher
     σ::Real = 1.0
     μ₀::Real = 0.0
-    λ::Real = 1.0
+    λ::Real = 1.0                                                              # regularization parameter
```

Comment:
In a sidenote in https://www.paltmeyer.com/blog/posts/bayesian-logit/ PA equates `lambda == 1 / sigma^2`. Is this true in our case as well?

Reply:
I believe not, but I am unsure, we can check with PA next week. One of the reasons why is that they can be instantiated with different values in the _LaplaceParams_ constructor, which wouldn't be the case if there was an equivalence relation between them. If there is indeed such a relation between them, we should change the constructor to reflect that.